charts/fluent-bit/values.yaml

```diff
@@ -73,6 +73,10 @@ serviceMonitor:
   # selector:
   #  prometheus: my-prometheus
 
+dashboards:
+  enabled: false
+  label: grafana_dashboard
```

Comment:
```suggestion
  labels:
    grafana_dashboard: "1"
```

Thanks!

nit: I think this with the corresponding change in the template will keep doors open for additional dashboard labels if ever needed, also label entails both key and value

Reply:
@naseemkullah are you sure that this is desired? The usual pattern for these is to provide a single label option to map to the single label that you provide to the sidecar, the value should be `1`. The config map will currently get all of the standard chart labels so I'd advise against this.

Reply:
the main goal is to be able to adjust the label key, this allows for that in a highly flexible manner, but with the increased flexibility someone may be tempted to do something weird, i generally err on the side of flexibility though, however to make it strict I'd nit that the key be renamed  from `label` to `labelKey`

Reply:
Changed to `labelKey`.